src/DevBetterWeb.Infrastructure/PaymentHandler/StripePaymentHandler/StripePaymentHandlerSubscriptionService.cs

```diff
@@ -113,12 +140,20 @@ private DateTime GetStartDate(Subscription subscription)
 
     private Stripe.Subscription GetSubscription(string subscriptionId)
     {
-
       var subscription = _subscriptionService.Get(subscriptionId);
 
       return subscription;
     }
 
+    public string GetAssociatedProductName(string subscriptionId)
```

Comment:
Why not get the subscription, and then you have its product name with it?

Reply:
this method is to get from Stripe the product name to be assigned to the subscriptionPlanName in SubscriptionPlanDetails in SubscriptionPlan, which a DevBetter (Member)Subscription has. I'm getting a stripe subscription, but I only want to return the product name because I don't want to pass a stripe subscription all over the place